### PR TITLE
script.copacetic.helper 1.0.9

### PR DESCRIPTION
**Pull request by realcopacetic**
### Description
**1.0.9**
- Background slideshows from custom paths/playlists are now generated via the background fanart fetching service and available globally throughout the skin, via a window property. Previously this was done in-skin using a container so would not be available persistently across windows.
- Removed glitch in background slideshows causing them to fetch new fanarts too often
- Removed unnecessary conditions from background monitor
- Added toggle_addon action to script.
- Switched incorrect labels Enabled/Disabled for script enabler toggle function
- Reordered addon.xml extension points to update how addon is categorised in Kodi repository

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.copacetic.helper/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Auto detect text files and perform LF normalization
```
Comment (basrieter) [inline script.copacetic.helper/.gitattributes:1]: Exclude please

File: script.copacetic.helper/.gitignore [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Addon
```
Comment (basrieter) [inline script.copacetic.helper/.gitignore:1]: Exclude this file please

**Review by basrieter — CHANGES_REQUESTED** [review]
Please remove the git files. Will look into it after that.

**Comment by realcopacetic** [comment]
Ahh sorry about that. I've deleted those files

**Review by basrieter — APPROVED** [review]

**Pull request merged**